### ISSUE TITLE
Code Sign error when using tns run ios --emulator

### ISSUE THREAD

**Issue by dapriett** [NativeScript/ios-runtime#488]
Hello,

I recently been unable to run my app using ios emulator

```
$ tns run ios --emulator

=== BUILD TARGET Pods OF PROJECT Pods WITH CONFIGURATION Debug ===

Check dependencies

=== BUILD TARGET demo OF PROJECT demo WITH CONFIGURATION Debug ===

Check dependencies
Code Sign error: No matching provisioning profiles found: No provisioning profiles with a valid signing identity (i.e. certificate and private key pair) matching the bundle identifier “org.nativescript.demo” were found.

** BUILD FAILED **


The following build commands failed:
        Check dependencies
(1 failure)
Command xcodebuild failed with exit code 65
```

Any clue why I would receive this error?  I shouldn't need to sign my code with the emulator, right?

Thanks!

**Comment by dapriett** [comment]
Tried running from xcode, I see this:

```
ld: warning: directory not found for option '-F/Projects/demo/platforms/ios/../../lib/iOS'
ld: framework not found Pods
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by fealebenpae** [comment]
Hi @dapriett,

We have seen issues where xcodebuild under Xcode 7.2 fails to codesign simulator builds which it shouldn't try to do in the first place. You can either downgrade to Xcode 7.1, or use the Xcode GUI. However, if you're using CocoaPods, be sure to open the Xcode workspace rather than the project under `platforms/ios`.

**Comment by dapriett** [comment]
Thanks - downgrading to Xcode 7.1 worked.

**Comment by NathanaelA** [comment]
@fealebenpae this issue probably should remain open as it isn't solved.    Since both 7.2 & 7.3 (Beta) have this same issue "we" as a community may need to figure out how to solve this without downgrading...     

One thing that appears to be common that I've seen is this issue only appears when adding a cocoapod.

**Comment by fealebenpae** [comment]
@NathanaelA at the runtime level there is only ever so much we can do to influence the build - I would suggest posting this on the CLI repo as it's the CLI that drives both Xcode and CocoaPods.

**Comment by NathanaelA** [comment]
Ah, you are correct, for some reason I though I was on the cli repo.  Sorry about that.   Too many repo screens open.  :grinning:

**Comment by NathanaelA** [comment]
@dapriett there is a fix for this now; see #1428 I meant to tag you in issue #1428 but forgot to; so I'll post a comment here.  :grinning:

**Comment by dapriett** [comment]
Awesome, thanks @NathanaelA!
